sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -154,6 +161,27 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
       settingKey[Map[String, util.JSON]]("Additional option to include in the generated 'package.json'")
 
     /**
+      * List of entry bundles to generate. By default it generates just one bundle
+      * for your main class.
+      *
+      * @group tasks
+      */
+    val webpackEntries: TaskKey[Seq[(String, File)]] =
+      taskKey[Seq[(String, File)]]("Webpack entry names and corresponding scala.js outputs to process with webpack")
+
+    val webpackEntryPoints: TaskKey[Seq[(String, File)]] =
+      taskKey[Seq[(String, File)]]("Webpack entry names and corresponding entrypoint files to process with webpack")
+
+    val webpackLibraries: TaskKey[Seq[(String, File)]] =
+      taskKey[Seq[(String, File)]]("Webpack entry names and corresponding library files after processing")
+    
+    val webpackLoaders: TaskKey[Seq[(String, File)]] =
+      taskKey[Seq[(String, File)]]("Webpack entry names and corresponding loader files after processing")
+
+    val webpackBundles: TaskKey[Seq[(String, File)]] =
+      taskKey[Seq[(String, File)]]("Webpack entry names and corresponding bundle files after processing")
```

Comment:
So are you suggesting keeping the keys but just not exposing them? I'm certainly ok with that. I introduced all the tasks in order to have a "flow" of files that were generated by the pipeline, I'm less excited about giving that aspect of it up.